Nos/Views/Moderation/FlagSuccessView.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+
+/// Shows a success message after the user has successfully flagged a user or content
+struct FlagSuccessView: View {
+    var body: some View {
+        VStack(spacing: 30) {
+            Image.circularCheckmark
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(height: 116)
+
+            Text(String(localized: .localizable.thanksForTag))
+                .foregroundColor(.primaryTxt)
+                .font(.clarity(.regular, textStyle: .title2))
+                .padding(.horizontal, 62)
```

Comment:
I think I'd just make sure this is centered and give it a smaller amount of padding, like maybe 20 or even 30 (I'm not sure if we have a standard amount). I'd do the same below, too, on the keepOnHelpingUs text. This would ensure it still looks great on smaller phones or iPad Slide Over.